diff --git a/src/XIVLauncher.Core/Components/MainPage/MainPage.cs b/src/XIVLauncher.Core/Components/MainPage/MainPage.cs
index d538997..583ce9a 100644
--- a/src/XIVLauncher.Core/Components/MainPage/MainPage.cs
+++ b/src/XIVLauncher.Core/Components/MainPage/MainPage.cs
@@ -730,11 +730,11 @@ public class MainPage : Page
                 return null!;
 
             if (App.Settings.RB_App1Enabled.Value && !string.IsNullOrWhiteSpace(App.Settings.RB_App1))
-                Program.CompatibilityTools.RunInPrefix("\"" + App.Settings.RB_App1 + "\"" + App.Settings.RB_App1Args, wineD3D: App.Settings.RB_App1WineD3D.Value);
+                Program.CompatibilityTools.RunInPrefix("\"" + App.Settings.RB_App1 + "\" " + App.Settings.RB_App1Args, wineD3D: App.Settings.RB_App1WineD3D.Value);
             if (App.Settings.RB_App2Enabled.Value && !string.IsNullOrWhiteSpace(App.Settings.RB_App2))
-                Program.CompatibilityTools.RunInPrefix("\"" + App.Settings.RB_App2 + "\"" + App.Settings.RB_App2Args, wineD3D: App.Settings.RB_App2WineD3D.Value);
+                Program.CompatibilityTools.RunInPrefix("\"" + App.Settings.RB_App2 + "\" " + App.Settings.RB_App2Args, wineD3D: App.Settings.RB_App2WineD3D.Value);
             if (App.Settings.RB_App3Enabled.Value && !string.IsNullOrWhiteSpace(App.Settings.RB_App3))
-                Program.CompatibilityTools.RunInPrefix("\"" + App.Settings.RB_App3 + "\"" + App.Settings.RB_App3Args, wineD3D: App.Settings.RB_App3WineD3D.Value);
+                Program.CompatibilityTools.RunInPrefix("\"" + App.Settings.RB_App3 + "\" " + App.Settings.RB_App3Args, wineD3D: App.Settings.RB_App3WineD3D.Value);
 
             App.StartLoading(Strings.StartingGame, Strings.HaveFun);
 
